lib/src/package_config_impl.dart

```diff
@@ -77,7 +77,7 @@ class SimplePackageConfig implements PackageConfig {
       var name = package.name;
       if (packageNames.contains(name)) {
         onError(PackageConfigArgumentError(
-            name, "packages", "Duplicate package name"));
+            name, "packages", "Duplicate package $name"));
```

Comment:
I'd do `"Duplicate package name '$name'"`. It's not a duplicate package (well, maybe it is, maybe it isn't, but let's not assume).

Reply:
Done.